docs/guide/copter/guided_mode.rst

```diff
@@ -303,9 +303,10 @@ Send the `MAV_CMD_DO_SET_ROI <http://copter.ardupilot.com/common-mavlink-mission
         vehicle.send_mavlink(msg)
         vehicle.flush()
 
-The ROI is "reset" when the mode or the method used to control movement is changed. If you've set a yaw or region-of-interest value then this will be set to the default (vehicle faces the direction of travel).
+From Copter 3.2.1 you can explicitly reset the ROI by sending the `MAV_CMD_DO_SET_ROI <http://copter.ardupilot.com/common-mavlink-mission-command-messages-mav_cmd/#mav_cmd_do_set_roi>`_ command with zero in all values. The vehicle will then face the direction of travel.
```

Comment:
This was still a bit unclear; I would propose text that addresses:

* "Available since version X" reads less ambiguously than "From", and is more familiar (i.e. Python docs)
* Is "all values" correct, as in lat/lon/alt also need to be 0? Is it more correct to say "all params"?
* "vehicle will then..." seems to need an active verb

And some proposed text:

```
*Available since Copter 3.2.1:* you can explicitly reset the ROI by sending the [`MAV_CMD_DO_SET_ROI`](http://copter.ardupilot.com/common-mavlink-mission-command-messages-mav_cmd/#mav_cmd_do_set_roi) command with zero in all parameters. The vehicle will return to face the direction of travel.
```